uisdk/src/main/java/com/karhoo/uisdk/screen/booking/map/BookingMapView.kt

```diff
@@ -388,15 +388,15 @@ class BookingMapView @JvmOverloads constructor(context: Context,
     fun setNoBottomPadding() {
         googleMap?.setPadding(0, 0, 0, 0)
         recentreMapIfDestinationIsNull()
+        showLocateUserButton()
```

Comment:
We could have a single method which hides and show the button

For example fun showLocationButton(show: Boolean) {
if(show) {

} else {
}
}

That way we optimize the number of methods both in the view and contract.

Reply:
I don't really get what you mean here?

Reply:
Nevermind. I got it now